frontend/public/locales/fr/learn.json

```
@@ -28,11 +28,11 @@
       "body": "Soyez plus confiant dans les choix que vous faites pour votre retraite. Utilisez ces ressources pour mieux comprendre les options qui s'offrent à vous et l'effet sur vos pensions publiques.",
       "cards": {
         "from-work-to-retirement": {
-          "body": "Vous pouvez choisir différentes manière d'effectuer une transition du monde du travail a celui de la retraite. Prendre sa retraite du monde du travail et commencer a recevoir ses pensions publiques sont deux choses différentes.",
+          "body": "Vous pouvez choisir différentes manières d'effectuer une transition du monde du travail à celui de la retraite. Prendre sa retraite du monde du travail et commencer à recevoir ses pensions publiques sont deux choses différentes.",
           "title": "Transition du monde du travail à la retraite"
         },
         "rules-of-thumb": {
-          "body": "Jetez un coup d'œil sur ces conseils pour vous aider a prendre la décision de quand commencer à recevoir vos pensions. Faite le choix adapter à votre situation.",
+          "body": "Jetez un coup d’œil à ces conseils pour vous aider à choisir quand commencer à recevoir vos pensions publiques. Faites le choix adapté à votre situation.",
```

```suggestion
          "body": "Jetez un coup d'œil à ces conseils pour vous aider à choisir quand commencer à recevoir vos pensions publiques. Faites le choix adapté à votre situation.",
```

(to remain consistent with the rest of the application)